Jenkinsfile.groovy

```diff
@@ -54,8 +46,7 @@ pipeline {
 		}
 		stage('Deploy') {
 			environment {
-				def VERSION = 'latest'
-				def PUBLISH_DIR = "Z:\\\\releases\\\\MetriCam2\\\\git\\\\${VERSION}\\\\"
+				def PUBLISH_DIR = "Z:\\\\releases\\\\MetriCam2\\\\git\\\\${GITHUB_BRANCH_NAME}\\\\"
```

Comment:
We wanted to remove the "git" folder and replace it with ".unstable" - unless the source branch is "stable"
-> goes into issue